app/src/main/res/layout/fragment_lots.xml

```diff
@@ -59,5 +59,21 @@
         app:layout_constraintTop_toBottomOf="@id/text_header_lot_list">
     </androidx.recyclerview.widget.RecyclerView>
 
-
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/fab"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:clickable="true"
+        android:contentDescription="@string/submit"
+        android:focusable="true"
```

Comment:
By default, this is "auto" which lets the framework determine whether a user can move focus to a view.
I think in your case you don't need this, because I have notice that when you run the app, when you touch the fab it does not work at first time.
I recommend delete this line, otherwise I would like to know why you're using it.

Reply:
the line goes